src/row.js

```diff
@@ -275,21 +249,21 @@ Row.formatFamilies_ = function(families, options) {
  *   var apiResponse = data[0];
  * });
  */
-Row.prototype.create = function(entry, callback) {
+Row.prototype.create = function(options, callback) {
```

Comment:
Should we allow users to use the old `row.create(rowData)` syntax?

Reply:
This would prevent the `gaxOptions` argument, because we can't have two consecutive, optional object arguments. I'm fine with not allowing `gaxOptions` here, although I believe that would be our first deviation, and could come back to bite us if users find they needed that type of control.

Reply:
Oh I meant doing some testing to see if options is a rowData object then treating it like this:

```js
if (isRowData(options) {
  options = {entry: options}
}
```

I'm not sure how to implement `isRowData` though. Something similar can also be done for other methods

Reply:
> I'm not sure how to implement isRowData though.

I think that's the problem I meant to illustrate as well, since both options and entries would be optional POJOs. If we special-case `gaxOptions` and `entry`, then we would be blocking row data where they meant to use those property names as part of their entry:

```js
row.create({ name: '1b', entry: true })
// isRowData() returns false
```

Reply:
Got it, makes sense